src/runtime/lpython_builtin.py

```diff
@@ -1085,6 +1085,64 @@ def _lpython_str_isspace(s: str) -> bool:
             return False
     return True
 
+@overload
+def _lpython_str_center(s: str, width: i32, fillchar: str) -> str:
+    """
+    Return centered in a string of length width. 
+    Padding is done using the specified fillchar (default is an ASCII space). 
+    The original string is returned if width is less than or equal to len(s).
+    """
+    if(len(fillchar) != 1):
+        raise TypeError("The fill character must be exactly one character long")
```

Comment:
Considering https://github.com/lcompilers/lpython/issues/2666, I think this will not print the error message. But I think it is fine for now. The error message should start printing as soon as we fix https://github.com/lcompilers/lpython/issues/2666.